app/models/sub_service_request.rb

```diff
@@ -368,13 +368,24 @@ def arms_editable?
     !self.in_work_fulfillment?
   end
 
-  # Callback which gets called after the ssr is saved to ensure that the
-  # past status is properly updated.  It should not normally be
-  # necessarily to call this method.
-  def update_past_status(identity)
-    old_status = self.past_statuses.last
-    if @prev_status and (not old_status or old_status.status != @prev_status)
-      self.past_statuses.create(status: @prev_status, date: Time.now, changed_by_id: identity.id)
+  # TODO: Verify that this method is no longer needed or being used
+  def candidate_statuses
```

Comment:
I just did a grep search and this method isn't called anywhere in the app. Probably safe to remove.